### DISCUSSION TITLE
A guide on how to ask a question

### DISCUSSION THREAD

**Discussion by lvjr** [lvjr/tabularray#120] [General]
Welcome to `tabularray` Discussions! Here is a guide for you to ask a question:

1. Please use English only to make more people understand your question.
2. Please always include some minimal but compilable code in your question.
3. Please always add triple backticks before and after your example code.

The following is a template for your to write a minimal example:

    ```latex
    \documentclass{article}
    
    \usepackage{xcolor}
    \usepackage{tabularray}
    
    \begin{document}
    
    \begin{tblr}{lcr}
    \hline
      Alpha   & Beta  & Gamma  \\
    \hline
      Epsilon & Zeta  & Eta    \\
    \hline
      Iota    & Kappa & Lambda \\
    \hline
    \end{tblr}
    
    \end{document}
    ```

**Comment by joanclario** [comment]
Dear all,

It turned out that I just recently discovered the package Tabularray and it has been very useful for my purposes. Thanks a lot. 

But now I have a question, since I am elaborating some very long tables and I don't want to page break them:
Is it possible to combine Tabularray to produce html format so that I can publish the (long) tables on the Web? 

Thanks in advance, --lario (ups, Catalonia)

**Reply by lvjr** [reply]
It is not a correct way to ask new questions in the comments. You need to click "New discussion" button to   ask a new question.

For html output, you may try tex4ht (see https://www.ctan.org/pkg/tex4ht).
